previous-projects.md

```diff
@@ -12,25 +12,19 @@ intro:
 
 {% include feature_row id="intro" type="center" %}
 <link rel="stylesheet" href="./assets/custom.css">
-<div class="container-projects">
```

Comment:
Any reason for removing this?

Reply:
It was an unused class that doesn´t exist anymore